metadata.py

```diff
@@ -42,15 +43,15 @@ def appmetadata() -> AppMetadata:
     )
     # and then add I/O specifications: an app must have at least one input and one output
     metadata.add_input(DocumentTypes.Document)
-    metadata.add_output(AnnotationTypes.Thing, typeSpecificProperty='property-value')
+    metadata.add_output(AnnotationTypes.NamedEntity, typeSpecificProperty='category=DATE')
```

Comment:
the property information should be just `category="DATE"` (https://sdk.clams.ai/autodoc/clams.appmetadata.html#clams.appmetadata.AppMetadata.add_input, any additional arguments are kwarg)